CMakeLists.txt

```diff
@@ -72,7 +78,7 @@ if (NOT "${ARCH}" STREQUAL "")
   endif()
 endif()
 
-if(WIN32 OR ARM7 OR ARM6)
+if(WIN32 OR ARM7 OR ARM6) #ARM8 may benefit from vectorisation
   set(OPT_FLAGS_RELEASE "-O2")
```

Comment:
Problem is I can't run the unit tests because of the gtest framework refusing to install on my system. Have opened another issue about this - all flavours of bitcoin run unit tests perfectly well on the same machine so what did they do differently?

I tried cloning googletest 1.7.0 from source, building as release, placing all files in the right places and still nada :(

So if someone else could run the unit tests that would be groovy...